protocol/consistencychecks.go

```diff
@@ -199,8 +195,8 @@ func (cc *ConsistencyChecks) verifyRegistration(msg *Response,
 func (cc *ConsistencyChecks) verifyKeyLookup(msg *Response,
 	uname string, key []byte) error {
 	df := msg.DirectoryResponse.(*DirectoryProof)
-	ap := df.AP
-	str := df.STR
+	ap := df.AP[0]
+	str := df.STR[0]
```

Comment:
`message.validate()` only checks that the lengths are > 0. I was asking about an additional registration- and lookups-specific validation that checks the lengths are == 1.